cvat/apps/iam/permissions.py

```diff
@@ -2025,6 +2025,8 @@ def check_permission(self, request, view, obj) -> bool:
         for perm in permissions:
             result = perm.check_access()
             allow &= result.allow
+            if not allow:
+                break
```

Comment:
I don't think the `allow` variable is needed at all, given this change. You can just replace this with:

```python
if not result.allow:
    return False
```

and the last line with

```python
return True
```

Reply:
agree

Reply:
In fact... I don't think the `permissions` list is needed either. You can merge this loop with the previous loop, and just check each permission as it is created.